-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-47679/44215

ERROR: Build errors found during clang-tidy run.
```
src/RecoParticleFlow/PFClusterProducer/src/PFClusterEMEnergyCorrector.cc:15:41: error: no matching constructor for initialization of 'const EcalReadoutTools::ESGetTokens' [clang-diagnostic-error]
   15 |     : ecalClusterToolsESGetTokens_{cc}, ecalReadoutToolsESGetTokens_{conf, cc}, calibrator_(new PFEnergyCalibration) {
      |                                         ^                           ~~~~~~~~~~
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02882/el8_amd64_gcc12/cms/cmssw/CMSSW_15_1_X_2025-03-24-2300/src/CalibCalorimetry/EcalTPGTools/interface/EcalReadoutTools.h:19:5: note: candidate constructor not viable: expects an rvalue for 2nd argument
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/CalibCalorimetry/EcalTPGTools/interface/EcalReadoutTools.h
+++ b/CalibCalorimetry/EcalTPGTools/interface/EcalReadoutTools.h
@@ -16,7 +16,7 @@ private:
 
 public:
   struct ESGetTokens {
-    ESGetTokens(const edm::ParameterSet&, edm::ConsumesCollector&& iC)
+    ESGetTokens(const edm::ParameterSet&, edm::ConsumesCollector iC)
         : ecalTrigTowerConstituentsMapToken{iC.esConsumes()}, ecalElectronicsMappingToken{iC.esConsumes()} {}
     edm::ESGetToken<EcalTrigTowerConstituentsMap, IdealGeometryRecord> const ecalTrigTowerConstituentsMapToken;
     edm::ESGetToken<EcalElectronicsMapping, EcalMappingRcd> const ecalElectronicsMappingToken;

